src/coreclr/src/jit/compiler.cpp

```diff
@@ -1975,6 +2061,21 @@ unsigned Compiler::compGetTypeSize(CorInfoType cit, CORINFO_CLASS_HANDLE clsHnd)
     return sigSize;
 }
 
+bool Compiler::compMethodIsNativeInstanceMethod(CORINFO_METHOD_INFO* mthInfo)
+{
+    return mthInfo->args.getCallConv() == CORINFO_CALLCONV_THISCALL;
+}
+
+CorInfoUnmanagedCallConv Compiler::compMethodInfoGetUnmanagedCallConv(CORINFO_METHOD_INFO* mthInfo)
+{
+    CorInfoCallConv callConv = mthInfo->args.getCallConv();
+    if (callConv == CORINFO_CALLCONV_DEFAULT || callConv == CORINFO_CALLCONV_VARARG)
+    {
+        return CORINFO_UNMANAGED_CALLCONV_UNKNOWN;
```

Comment:
I'm currently using `CORINFO_UNMANAGED_CALLCONV_UNKNOWN` as the representation for "this is a managed calling convention". Since both `CORINFO_CALLCONV_DEFAULT` and `CORINFO_CALLCONV_VARARG` are variations on the managed calling convention, I had them both map to `CORINFO_UNMANAGED_CALLCONV_UNKNOWN`.